channels/it.m3u

```diff
@@ -450,6 +454,8 @@ https://senato-live.morescreens.com/SENATO_1_001/playlist.m3u8
 https://senato-live.morescreens.com/SENATO_1_001/playlist.m3u8?video_id=13440&uuid=&channel_name=senato_webtv1&detected_delivery_method=hls
 #EXTINF:-1 tvg-id="SophiaTV.it" tvg-name="Sophia TV" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/RPXusvS.png" group-title="",Sophia TV (720p)
 https://www.onairport.live/sophiatv-it-live/livestream_low/playlist.m3u8
+#EXTINF:-1 tvg-id="Spike.it" tvg-name="Spike" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/og6Q9rj.png" group-title="",Spike (480p)
```

Comment:
I had the playlist on request, need to see if it works using VLC.

Reply:
Sure, all channel added in this request, works fine in Italy on VLC and Kodi. I've tested before put it in the list. :3

Reply:
Just tested, works for me.

Reply:
Spike is also possessed by Viacom Italia. Viacom is American-based.

Reply:
Yes but it's a italian version of Spike. Like VH1 or Paramount Network. Indeed sometime on-air are present italian movie also. 

https://www.paramountnetwork.it/spike

Reply:
Yes, but we tend to sort the channels by their origin. Since Paramount Network, VH1 and Spike all come from Viacom Enterprises, an American company, they should be put on the "us.m3u" category.

Of course, you can add "(Italy)" next to the name of the channel once put on the US playlist, to notify the end user that this channel is the Italian variant.

Reply:
Well, these are italian tv channel on-air only in Italy and someone like Paramount Channel (propriety of Viacom Italy this one too) is geo-blocked out of Italy, because visible only in Italy... And goes in US list? Sorry, but no sense. Because Viacom Italia have live channels (Paramount Network, Spike, VH1 and Super!) on-air via antennas from Italy. :/

Reply:
> 
> 
> Well, these are italian tv channel on-air only in Italy and someone like Paramount Channel (propriety of Viacom Italy this one too) is geo-blocked out of Italy, because visible only in Italy... And goes in US list? Sorry, but no sense. Because Viacom Italia have live channels (Paramount Network, Spike, VH1 and Super!) on-air via antennas from Italy. :/

https://www.lyngsat.com/tvchannels/us/Paramount-Network-Italia.html
https://www.lyngsat.com/tvchannels/us/Spike-Italia.html
https://www.lyngsat.com/tvchannels/us/VH1-Italia.html

All three are classified within the United States. They *might* stream in Italy, but they're not Italian *by nature*. 
Those are simply American channels having their proper Italian variant for Italian people.

Reply:
The channels name are american, ok, but Viacom Italy have his headquarters at Milan in italy. They broadcast from Milan, sometimes italian shows with italian celebrity. Example "Battistology" on Spike Italia. And Battistology not is an american tv show.

https://it.wikipedia.org/wiki/ViacomCBS_Networks_Italia
https://www.paramountnetwork.it/programmi/h2qb3z/battistology

But ok, i don't care, reject the question. No problem. Bye. :3

Reply:
> 
> 
> The channels name are american, ok, but Viacom Italy have his headquarters at Milan in italy. They broadcast from Milan, sometimes italian shows with italian celebrity. Example "Battistology" on Spike Italia. And Battistology not is an american tv show.
> 
> https://it.wikipedia.org/wiki/ViacomCBS_Networks_Italia
> https://www.paramountnetwork.it/programmi/h2qb3z/battistology
> 
> But ok, i don't care, reject the question. No problem. Bye. :3

Hmm...

@freearhey what do you think ?

Reply:
If you don't know exactly which playlist to add a channel to, it's better to refer to the channel description on lyngsat. The point is that we break channels into files only to make it easier for us to find them later, and in this case it will be easier if all "Spike" channels are stored in the US playlist.

Reply:
Well all of them says "US" and "Italy" at the same instance...

Reply:
I said this because only the US is in the description of each version of the channel:
https://www.lyngsat.com/tvchannels/us/Spike-Italia.html
https://www.lyngsat.com/tvchannels/us/Spike-Rossiya.html
https://www.lyngsat.com/tvchannels/us/Spike-Ukraina.html